knimin/templates/pm_sequence.html

```diff
@@ -1,77 +1,72 @@
 {% extends logged_in_index.html %}
 {% block head %}
 {% from json import dumps %}
-<script>
-  /**
-   *
-   * Checks if the current value of the element exists in the given array
-   * and updated its style accordingly
-   *
-   * @param {Element} elem The input DOM element
-   * @param {Array} existing The array of existing elements
-   *
-   **/
-  function formatInput(elem, existing) {
-    var val = $(elem).val().trim();
-    $(elem).removeClass('pm-warning');
-    if (val.length > 0 && $.inArray(val, existing) === -1) {
-      $(elem).addClass('pm-warning');
-    }
-  };
-
-  /**
-   *
-   * Updates the kit lots values based on the current value of reagent type
-   **/
-  function updateKitLotsValues () {
-    var reagents = {% raw dumps(reagents) %};
-    var currVals = reagents[$('#reagent-kit-type').val()];
-    $(".autocomplete-reagent").autocomplete({source: currVals,
-                                             select: function (e, ui) {
-                                               formatInput(this, currVals)
-                                             }});
-    $(".autocomplete-reagent").keyup(function(e) {
-      formatInput(this, currVals);
-    });
-  };
-
-  $(document).ready(function() {
-    updateKitLotsValues();
-    $("#reagent-kit-type").change(function() {
-      updateKitLotsValues();
-    });
-  });
-</script>
+<script></script>
 {% end %}
 {% block content %}
 <h3>Prepare sequencing run for sample pool: {{pool['name']}} (ID: {{pool['id']}})</h3>
 <div class='container'>
   <form action="/pm_sequence/" method="post">
-    <input type="hidden" name="pool_id" value="{{pool['id']}}"/>
-    <div class='row'>
+    <div class="row">
+      <!-- Platform -->
+      <div class='col-xs-4 form-group'>
+        <label>Platform:</label>
+        <select name="platform" class="form-control" required>
+          {% for p in ['Illumina'] %}
+            <option value='{{p}}'>{{p}}</option>
+          {% end %}
+        </select>
+      </div>
+      <!-- Instrument model -->
       <div class='col-xs-4 form-group'>
-        <label>Sequencer:</label>
-        <select name="sequencer" class="form-control" required>
-          {% for seq in sequencers %}
-            <option value='{{seq['id']}}'>{{seq['name']}} ({{seq['platform']}}, {{seq['instrument_model']}})</option>
+        <label>Instrument model:</label>
+        <select name="instrument_model" class="form-control" required>
+          {% for im in ['MiSeq', 'HiSeq 2500', 'HiSeq 2500 Rapid Run', 'HiSeq 4000'] %}
```

Comment:
@antgonza help me figuring this out. Apparently the enums where created by default on the `ag` schema and that's why I couldn't retrieve it. @antgonza is performing other changes on the DB and to avoid merge conflicts he will make the changes so these enums get created on the `pm` schema. I will then update this to reflect that change.